windows/v.mod

```diff
@@ -0,0 +1,6 @@
+Module {
+        name: 'windows'
+		description: 'shcore.dll',
+        version x64: Windows Kits\10\Lib\10.0.18362.0\um\x64
+		version x86: Windows Kits\10\Lib\10.0.18362.0\um\x86
```

Comment:
What is the license for the binaries (`x64/shcore.lib` and `x86/shcore.lib`) you're bundling with this PR?

Reply:
See Microsoft docs in windows_shcore folder:
  (1):  Microsoft sdk_license.pdf
  (2):  Microsoft Windows SDK redist lists _ Microsoft Docs.pdf

As seen in REDIST.TXT (2)
.
.
Static LIB files
Subject to the license terms for the software, the .lib files under the following directories may be distributed unmodified when built as part of your program:
Program Files\Windows Kits\10\Lib<version>\um\x86
Program Files\Windows Kits\10\Lib<version>\um\x64
.
.
As shcore.dll  is part of the above directories, I think it is distributable, but there are a lot of distribution requirements see in (1) 2.a.i. and ii.